debian/patches/series

```diff
@@ -13,3 +13,7 @@
 0013-Fix-up-perl-path-in-installed-scripts.patch
 0017-python-oneliner-for-realpath.patch
 0018-run-automake.patch
+0001-golang-return-error-interface-instead-of-struct.patch
```

Comment:
The numbering here seems wrong. Shouldn't these be 0019-0022?

Reply:
so, @ajames can probably answer this better, but these are actually combined using quilt.  Basically, I created the patch files using git format-patch based on 4 commits I made on top of the 1.38.6 version of libguestfs.  You'll notice there are already numbers skipped and duplicated in the patch list (thought not too many).  (i.e. there are two patches prefixed with 0013 already)